### ISSUE TITLE
contents of files/pkg-message.in is not extracted

### ISSUE THREAD

**Issue by dlangille** [FreshPorts/freshports#117]
See how this `PKGMESSAGE` value is in the working directory?

```
[dan@dev-ingress01:/var/db/freshports/ports-jail/var/db/repos/PORTS-head/ports-mgmt/pkg-provides] $ make -V PKGMESSAGE
/var/ports/var/db/freshports/ports-jail/var/db/repos/PORTS-head/ports-mgmt/pkg-provides/work/pkg-message
```

That is not available without building the package....

Thus, https://www.freshports.org/ports-mgmt/pkg-provides/ does not show pkg-mesage information, but https://www.freshports.org/sysutils/rubygem-bolt/ can.

I wonder if there is a way to get `work/pkg-message` created.

**Comment by dlangille** [comment]
This seems to be sufficient:

```
$ make patch
$ make apply-slist
$ ls -l /var/ports/var/db/freshports/ports-jail/var/db/repos/PORTS-head/sysutils/bacula9-server/work/pkg-message.server
-rw-r--r--  1 root  wheel  1746 Jun 10 15:12 /var/ports/var/db/freshports/ports-jail/var/db/repos/PORTS-head/sysutils/bacula9-server/work/pkg-message.server
```

This is far from ideal. It requires fetching the `DISTFILES`.  It requires patching.  It probably also requires building of dependencies.

**Comment by dlangille** [comment]
I could:

* grep PKGMESSAGE for `/work/`
* if found, set port.pkgmessage = 'This package does not have a simple pkg-message file'

**Comment by dlangille** [comment]
This may be solved by #245

**Comment by dlangille** [comment]
Seems good now.
